Gems/GenAIFramework/Code/Include/GenAIFramework/GenAIFrameworkTypeIds.h

```diff
@@ -13,9 +13,19 @@ namespace GenAIFramework
     // System Component TypeIds
     inline constexpr const char* GenAIFrameworkSystemComponentTypeId = "{C1447E9E-C102-4FDA-8D62-34BEE2DD4D31}";
     inline constexpr const char* GenAIFrameworkEditorSystemComponentTypeId = "{CD496C8B-7D2A-447B-A1F9-B449A9BFEC04}";
+    inline constexpr const char* GenAIFrameworkSystemComponentConfigurationTypeId = "{6AB6E636-60DC-4377-BD45-2326CF6A0069}";
 
     inline constexpr const char* GenAIAsyncRequestSystemComponentTypeId = "{D2D195A2-C720-4961-9AEF-C00FCB0AA70D}";
-    inline constexpr const char* GenAIAsyncRequestEditorSystemComponentTypeId = "{018e7a45-43ae-7e69-8bd9-0751b9170bf5}";
+    inline constexpr const char* GenAIAsyncRequestEditorSystemComponentTypeId = "{018E7A45-43AE-7E69-8BD9-0751B9170BF5}";
+
+    // Communication TypeIds
+    inline constexpr const char* SystemRegistrationContextTypeId = "{E162937C-0177-4CED-87B3-3B037A44C394}";
+    inline constexpr const char* AsyncRequestsTypeId = "{018E7A12-5DB2-7DAA-A764-78D7703C8EFA}";
+    inline constexpr const char* AIServiceProviderRequestsTypeId = "{0BCAC4E2-09A9-4702-98DD-EF37006664A7}";
+    inline constexpr const char* AIModelRequestTypeId = "{8C140E10-E3A2-478B-9886-C7C24EB16816}";
```

Comment:
```suggestion
    inline constexpr const char* AIModelRequestsTypeId = "{8C140E10-E3A2-478B-9886-C7C24EB16816}";
```
Nit: changing the typeid name to be more consistent with AIModelRequests change